content/tooling/use-angular-dev-tools.md

```diff
@@ -0,0 +1,27 @@
+---
+title: use Angular DevTools
+author:
+  name: Maciej Wójcik
+  url: https://twitter.com/maciej_wwojcik
+---
+
+# Problem
+
+Understanding complex and nested components trees can be challenging, especially if you are interested in the DOM view
+and the logic and data layer. In addition, it's not easy to debug Angular components using standard debug tools because
+they are not adapted very well for Angular-specific features.
```

Comment:
```suggestion
Understanding complex and nested component trees can be challenging, especially if you are interested in the DOM view, the logic, and data layer. In addition, it's not easy to debug Angular components using standard debugging tools because they are not adapted very well for Angular-specific features.
```

Reply:
Suggested some rephrasing and grammatical issues.